apps/sensenet/src/components/login/Login.tsx

```diff
@@ -112,7 +111,7 @@ export const Login = () => {
         })
         if (match.path === '/login') {
           await sleepAsync(1800)
-          const { from } = location.state || { from: { pathname: `/${btoa(repoToLogin.configuration.repositoryUrl)}` } }
+          const { from } = { from: { pathname: `/${btoa(repoToLogin.configuration.repositoryUrl)}` } }
           history.replace(from)
```

Comment:
While this is going to be removed in #568 . We can merge this if you want. The desired behavior should be also in the new auth pr. 
```suggestion
          history.replace(`/${btoa(repoToLogin.configuration.repositoryUrl)}`)
```